barretenberg/cpp/src/barretenberg/vm2/constraining/recursion/goblin_avm_recursive_verifier.hpp

```diff
@@ -138,13 +138,14 @@ class AvmGoblinRecursiveVerifier {
         auto mega_verifier_output = mega_verifier.verify_proof(mega_proof);
 
         // Recursively verify the goblin proof\pi_G in the Ultra circuit
-        TableCommitments t_commitments = mega_verifier.key->witness_commitments.get_ecc_op_wires().get_copy();
-        TableCommitments T_prev_commitments =
+        MergeCommitments merge_commitments;
```

Comment:
You could rather use as it's a bit cleaner in places where you're creating the `InputCommitments` object  
```
MergeCommitments merge_commitments {
 .t_commitments = ... , 
 .T_prev_commitments = ... } 
```

Reply:
or  `merge_commitments = MergeCommitments{..}` in other cases